README.md

```diff
@@ -43,6 +43,35 @@ reminderbatch -w <WAVE_NUMBER> -b <STARTING_BATCH_NUMBER> -a <ACTION_PLAN_ID> --
 reminderbatch -w <WAVE_NUMBER> -b <STARTING_BATCH_NUMBER> -a <ACTION_PLAN_ID> --insert-rules --trigger-date-time=<DATE_TIME> --max-cases=1000000 
 ```
 
+## Response-driven reminder scheduling
+We can create response-driven action rules from a provided CSV file of LSOAs. 
+
+#### Get counts of cases for response-driven reminders
+To get a count of cases using the LSOAs file:
```

Comment:
How are we getting this file? I think we settled on using a bucket but I don't think a bucket exists yet for this purpose. We'd need a documented way of getting the file from the bucket into this service. (happy for this side to be on a separate ticket if we want but we need to keep track of it)

Reply:
I thought I saw a separate ticket for the bucket create. Will give the guys a shout to see what they want us to do with this

Reply:
Bucket is showing as 'TBC' in the file spec, so wonder if there is another ticket hiding somewhere.

Reply:
Had a quick discussion and a bucket will be created to allow the business area to drop the reminder files into. Updated the ticket with the new PR 👍